CONTRIBUTING.md

```diff
@@ -109,26 +109,23 @@ If developing with Eclipse and M2Eclipse (the Maven tooling for Eclipse), just l
 To use a local build of the `jib-gradle-plugin`:
 
   1. Build and install `jib-gradle-plugin` into your local `~/.m2/repository`
-     with `./gradlew jib-gradle-plugin:install`;
-  1. Modify your test project's `build.gradle` to look like the following:
+     with `./gradlew jib-gradle-plugin:publish`
+  1. Add to your test project's `settings.gradle` the following:
         ```groovy
-        buildscript {
-            repositories {
-                mavenLocal() // resolve in ~/.m2/repository
-                mavenCentral()
-            }
-            dependencies {
-                classpath 'com.google.cloud.tools:jib-gradle-plugin:2.1.1-SNAPSHOT'
-            }
+        pluginManagement {
```

Comment:
Can we add that this must be the first block in `settings.gadle`? https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_management

Reply:
yeah that's a good idea